src/UserAdmin/src/Smart.FA.Catalog.Web/Identity/UserIdentity.cs

```diff
@@ -7,26 +7,22 @@ namespace Smart.FA.Catalog.Web.Identity;
 /// <inheritdoc />
 public class UserIdentity : IUserIdentity
 {
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
     /// <inheritdoc />
-    public int Id => Identity.Id;
+    public int Id => Identity?.Id ?? 0;
```

Comment:
It's pointless to use the coalesce operator in that case since identity will never be null (and Id neither).
```suggestion
    public int Id => Identity.Id;
```

Reply:
When seeding it is equal to zero.